test/jdk/java/net/vthread/HttpALot.java

```diff
@@ -23,13 +23,20 @@
 
 /**
  * @test
+ * @bug 8294610
  * @summary Stress test the HTTP protocol handler and HTTP server
  * @requires vm.debug != true
  * @modules jdk.httpserver
  * @library /test/lib
  * @compile --enable-preview -source ${jdk.version} HttpALot.java
+ * @comment The test launch command intentionally uses -Dsun.net.httpserver.nodelay=true
```

Comment:
Maybe replace this "The test run with -Dsun.net.httpserver.nodelay=true" as the words "launch command intentionally uses" aren't really needed.

Reply:
Done. Updated the PR to use the suggested wording.